### ISSUE TITLE
Tensorboard tensor representation when a Op (or a node) has multiple output tensors

### ISSUE THREAD

**Issue by Choibae** [tensorflow/tensorflow#2144]
I'm using the op 'tf.train.shuffle_batch,' has two output tensors, feature and label.

When I visualize it on tensorboard, thickness of edge(tensor) looks weird.
The label tensor, to loss node, has thickness of the feature tensor.

It seems like current tensorboard visualizes thickness using first output tensor of the op for all out edges.
Is it possible? multiple output tensors has their own shape?

![screenshot from 2016-04-28 23 26 19](https://cloud.githubusercontent.com/assets/15023894/14878512/a15912fc-0d56-11e6-991a-f7445ff36823.png)

**Comment by dsmilkov** [comment]
Hi,

Thanks for filing and apologies for the delay in response. This was fixed with f12b843d78b0119b3f8d0ffc99a9e417d9232e7c. You should be able to see correct edge shapes (not only the first output shape) in the 0.9 release, or if you don't want to wait, we just published its release candidate https://github.com/tensorflow/tensorflow/releases/tag/v0.9.0rc0
